spec/Vehicle/Vehicle.vspec

```diff
@@ -191,6 +191,24 @@ AmbientAirTemperature:
   unit: celsius
   description: Ambient air temperature outside the vehicle.
 
+AmbientHumidity:
+  datatype: float
```

Comment:
Is the level of precision of float required here?

Would a uint8 not be sufficient?

Apologies if these kinds of questions have already been raised and addressed in your design review process.

Reply:
In some standards they use better precision than int. As an example, in J1939 PGN 65164 SPN 354 "Relative Humidity, Measures humidity of combustion air prior to entry into turbocharger", humidity is represented as 1 byte, with 0.4% per bit. If we would use this J1939 signal as source for this VSS signal (or in the other direction) both 249 (99.6%) and 250 (100%) would be represented as 100% in VSS, i.e. some precision would be lost.

Using float minimizes loss of precision when converting between e.g. VSS-representation and J1939-representation